src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import Navigation from './routes/Navigation';
+import BookList from './components/BookList';
+import BookForm from './components/BookForm';
+
+const App = () => {
+  const [books, setBooks] = React.useState([]);
+
+  const handleAddBook = (newBook) => {
+    setBooks((prevBooks) => [...prevBooks, newBook]);
+  };
+
+  const handleDeleteBook = (bookId) => {
+    setBooks((prevBooks) => prevBooks.filter((book) => book.id !== bookId));
+  };
+
+  return (
+    <Router>
+      <div>
+        <Navigation />
+        <Routes>
+          <Route path="/" element={<BookList books={books} onDelete={handleDeleteBook} />} />
+        </Routes>
```

Comment:
- [ ] Nice job so far, I would advise you to follow what the requirements say: Categories should also be in routes and not a placeholder in Navigation as you did it
![Capture](https://github.com/gdvl777/sample_bookstore/assets/63972670/55fda23d-9f4c-4ded-95a2-34028b291bee)